src/services/CommunityService.ts

```diff
@@ -30,8 +29,14 @@ const request = async (): Promise<string> => {
         feedUrl = '/nemflash';
     }
     // execute request
-    const response = await axios.get(feedUrl, { params: {} });
-    return response.data;
+    const response = await fetch(feedUrl, {
+        method: 'GET',
+    })
+        .then((response) => response.text())
```

Comment:
I'm not sure if axios's data is a text or a parsed json. Please double-check this. A unit test would do...

Reply:
Already checked axios response before changing it to use fetch and it was returning text, not json.